runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -179,40 +168,18 @@ default Color defaultColor()
 		keyName = "highlightedColor",
 		name = "Highlighted items color",
 		description = "Configures the color for highlighted items",
-		position = 12
+		position = 11
 	)
 	default Color highlightedColor()
 	{
 		return Color.decode("#AA00FF");
 	}
 
-	@ConfigItem(
-		keyName = "lowValueColor",
-		name = "Low value items color",
-		description = "Configures the color for low value items",
-		position = 13
-	)
-	default Color lowValueColor()
-	{
-		return Color.decode("#66B2FF");
-	}
-
-	@ConfigItem(
-		keyName = "lowValuePrice",
-		name = "Low value price",
-		description = "Configures the start price for low value items",
-		position = 14
-	)
-	default int lowValuePrice()
```

Comment:
Leaving only "Medium" "High" and "Insane" is weird without Low. I don't see why this hurts anyway keeping it?

Reply:
Because there is highlight > value instead of it now i guess